### DISCUSSION TITLE
Deformable part  in robots

### DISCUSSION THREAD

**Discussion by JunjieMa5515** [isaac-sim/IsaacLab#1510] [Q&A]
Isaaclab supports variable parts, but if my robot has rigid body part and deformable part, how should I define it? If I set up a rigid body collision network for the deformable part in the USD file, but link the physics of the network to the deformable part, it seems that it will still be a rigid body after importing it, is there any good way?

**Comment by StrainFlow** [comment]
Have you followed the tutorial on deformable objects? 

https://isaac-sim.github.io/IsaacLab/main/source/tutorials/01_assets/run_deformable_object.html

**Comment by yash-s20** [comment]
different discussion that might be relevant - https://github.com/isaac-sim/IsaacLab/discussions/1327
